src/distributions/adoptium/models.ts

```diff
@@ -0,0 +1,38 @@
+// Models from https://api.adoptium.net/q/swagger-ui/#/Assets/searchReleasesByVersion
```

Comment:
src directory name should change from src/distributions/adoptium to src/distributions/temurin